src/dataset.rs

```diff
@@ -67,3 +96,26 @@ struct Metadata {
     name: String,
     notes: String,
 }
+
+pub fn build_metadata(
+    timestamp: DateTime<Utc>, 
+    device: String, 
+    country: String, 
+    region: String,
+    site: String,
+    name: String,
+    notes: String) -> Metadata {
+        Metadata { timestamp, device, country, region, site, name, notes }
+    }
+
+pub fn build_mission_metadata(args: &InitMissionArgs) -> Metadata {
+    build_metadata(
+        args.timestamp.into(), 
+        args.device.clone(), 
+        args.country.clone(), 
+        args.region.clone(), 
+        args.site.clone(), 
+        args.name.clone(),
+        String::new()
```

Comment:
This currently doesn't take any data from the `notes` field.

Reply:
Just to clarify, does the `notes` field correspond to the `message` field of `InitMissionArgs`?

Reply:
I believe so, I forgot the names didn't match. This is also how they are defined in the Python codebase.